mailer/src/main/scala/io/scalac/newspaper/mailer/ChangeDetectedPBDeserializer.scala

```diff
@@ -2,15 +2,15 @@ package io.scalac.newspaper.mailer
 
 import java.util
 import org.apache.kafka.common.serialization.Deserializer
-import io.scalac.newspaper.mailer.events._
+import io.scalac.newspaper.events._
 
 case class ChangeDetectedPBDeserializer() extends Deserializer[ChangeDetected] {
   override def configure(configs: util.Map[String, _], isKey: Boolean): Unit = () // nothing to do
 
   override def close(): Unit = () // nothing to do
 
   override def deserialize(topic: String, data: Array[Byte]): ChangeDetected = {
-//    ChangeDetected.apply("foo", "bar", "baz") //TODO: hard to say if it works :)
+//    ChangeDetected.apply("foo", "bar") //in case of poisonous messages stored on kafka
```

Comment:
You can't trust other services and you have to protect yourself against problems/bugs in other services. Currently mailer doesn't do that. If `analyzer` (or any other service or command line publisher) publishes a buggy (non-parseable) message then mailer will explode and even if we restart it - it will explode again, effectively being "stuck". Until I have a better way to deal with this problem, I use this line to clean up kafka